ui/src/components/SampleControls/SnapshotControl.jsx

```diff
@@ -25,19 +25,30 @@ function SnapshotControl(props) {
     return sourceScale * imageRatio;
   });
 
-  const takeSnapshot = useCallback(() => {
+  const takeSnapshot = useCallback(async () => {
     const img = document.querySelector('#sample-img');
     const fimg = new fabric.Image(img);
     fimg.scale(imageRatio);
 
     canvas.setBackgroundImage(fimg);
     canvas.renderAll();
 
-    const filename = `${proposal}-${currentSampleName}.jpeg`;
-    download(filename, canvas.toDataURL({ format: 'jpeg' }));
+    const imgDataURI = canvas
+      .toDataURL({
+        format: 'png',
+        backgroundColor: null,
+      })
+      .split(',')[1];
 
     canvas.setBackgroundImage(0);
     canvas.renderAll();
+
+    const filename = `${proposal}-${currentSampleName}.jpeg`;
```

Comment:
We send PNG and receive JPEG in return. It's a bit counter-intuitive. Is there a particular reason?

Reply:
@walesch-yan, thats perhaps me who did something strange when I rebased on `esrf-develop`.  The code above: `format: 'png'` should perhaps have been `format: 'jpeg'`

Reply:
I think that was my mistake as I wrote the back-end code after the sending part and before the receiving part and did not pay attention to it in between sorry about that

Reply:
No worries!